notion/client.py

```diff
@@ -310,7 +310,7 @@ def search_pages_with_parent(self, parent_id, search=""):
         data = {
             "query": search,
             "parentId": parent_id,
-            "limit": 10000,
+            "limit": 100,
```

Comment:
I think it'd be better to store this as a constant somewhere. So if it changes later, there's only 1 place to update. Thoughts?

Reply:
maybe it could be an argument in NotionClient.__init__() ?